content/go/concepts/comments/comments.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: 'Comments'
+Description: 'A comment is a piece of text within a program that is not executed. It can be used to provide additional information to aid in understanding the code.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Comments'
+  - 'Documentation'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/back-end-engineer-career-path'
+  - 'paths/computer-science'
+---
+
+A comment is a piece of text within a program that is not executed. It can be used to provide additional information to aid in understanding the code.
+
+## Single-line Comments
+
+Single-line comments are created using two consecutive forward slashes. The compiler ignores any text after `//` on the same line.
+
+```go
+// This line will denote a single line comment in Go
+```
+
+## Multi-line Comments
+
+Multi-line comments are created using `/*` to begin the comment, and `*/` to end the comment. The compiler ignores any text in between.
+
+```go
+/*
+This is all commented out.
+None of it is going to run!
+*/
+```
+
+Multi-line comments can also be created using `//` to begin each line.
+
+```go
+//This is all commented out.
+//None of it is going to run!
+```
+
+## Doc Comment
+
+Doc comments are used to identify the author of the code, the date the code was written, a description of the program, etc. Multi-line comments can be used to create a doc comment at the top of the file.
```

Comment:
Instead of "etc" can we say "and other <describe the type of info here in your own words>."

Reply:
This comment has been fixed.